lntest/harness_net.go

```diff
@@ -998,6 +999,12 @@ type OpenChannelParams struct {
 	// ScidAlias denotes whether the channel will be an option-scid-alias
 	// channel type negotiation.
 	ScidAlias bool
+
+	// Channel fees applied during the channel announcement phase.
```

Comment:
I thought I already added this comment in my last review, but it looks like I forgot. But those struct fields all need an individual Godoc comment.